zorg/jenkins/jobs/jobs/lldb-statistics

```diff
@@ -0,0 +1,187 @@
+#!/usr/bin/env groovy
+
```

Comment:
Can you add a top-level comment along the lines of:

This pipeline collects LLDB performance data by by running LLDB benchmarks while debugging a stable version of clang-19. ...